<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ventana de Juego</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="fondo">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>480</width>
      <height>510</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/fondos/fondo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_izq">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>240</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 159, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_der">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>240</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 159, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="fondo_sup">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(199, 159, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="logo_vjuego">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>110</width>
      <height>110</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="titulo_tienda">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>200</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt; font-weight:600;&quot;&gt;Tienda&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Valor Pingüirin $&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dinero:&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="valor_dinero">
    <property name="geometry">
     <rect>
      <x>862</x>
      <y>320</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;1000&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="cancion">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Canción:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="dificultad">
    <property name="geometry">
     <rect>
      <x>599</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Dificultad:&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="progreso">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Progreso:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barra_progreso">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::DefaultContextMenu</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="aprobacion">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Aprobación:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barra_aprobacion">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="et_combo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>50</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Combo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="et_mayor_combo">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>90</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Mayor Combo:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="combo_actual">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="combo_mayor">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>30</y>
      <width>151</width>
      <height>100</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vbox_botones">
     <item>
      <widget class="QPushButton" name="boton_pausar">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Pausar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_salir_juego">
       <property name="text">
        <string>Salir</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_comenzar_partida">
       <property name="text">
        <string>Comenzar Partida</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>650</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>            A          W           S           D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>680</y>
      <width>240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>170</y>
      <width>240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>680</y>
      <width>240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>2</width>
      <height>510</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>480</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>680</y>
      <width>480</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>980</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>980</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>2</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>10</y>
      <width>2</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="captura_left">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 225, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="captura_up">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 225, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="captura_down">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 225, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="captura_right">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 225, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>240</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/flechas/left_5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/flechas/up_5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/flechas/down_5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>600</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>sprites/flechas/right_5.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="boton_cancion">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>48</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Canción 1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canción 2</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="boton_dificultad">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>86</y>
      <width>126</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Principiante</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aficionado</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Maestro</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>2</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>590</y>
      <width>2</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>590</y>
      <width>2</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>590</y>
      <width>2</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>590</y>
      <width>2</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>200</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>200</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="valor_pinguirin">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>250</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>500</string>
    </property>
   </widget>
   <zorder>fondo_sup</zorder>
   <zorder>fondo</zorder>
   <zorder>fondo_izq</zorder>
   <zorder>fondo_der</zorder>
   <zorder>logo_vjuego</zorder>
   <zorder>titulo_tienda</zorder>
   <zorder>valor_dinero</zorder>
   <zorder>cancion</zorder>
   <zorder>dificultad</zorder>
   <zorder>progreso</zorder>
   <zorder>barra_progreso</zorder>
   <zorder>aprobacion</zorder>
   <zorder>barra_aprobacion</zorder>
   <zorder>et_combo</zorder>
   <zorder>et_mayor_combo</zorder>
   <zorder>combo_actual</zorder>
   <zorder>combo_mayor</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_14</zorder>
   <zorder>label_17</zorder>
   <zorder>label_18</zorder>
   <zorder>label_19</zorder>
   <zorder>label_20</zorder>
   <zorder>label_21</zorder>
   <zorder>label_22</zorder>
   <zorder>label_23</zorder>
   <zorder>label_24</zorder>
   <zorder>label_25</zorder>
   <zorder>label_26</zorder>
   <zorder>label_27</zorder>
   <zorder>label_28</zorder>
   <zorder>label_29</zorder>
   <zorder>captura_up</zorder>
   <zorder>captura_down</zorder>
   <zorder>captura_right</zorder>
   <zorder>captura_left</zorder>
   <zorder>label_31</zorder>
   <zorder>label_16</zorder>
   <zorder>label_15</zorder>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>label_5</zorder>
   <zorder>boton_cancion</zorder>
   <zorder>boton_dificultad</zorder>
   <zorder>label_30</zorder>
   <zorder>label_32</zorder>
   <zorder>label_33</zorder>
   <zorder>label_34</zorder>
   <zorder>label_35</zorder>
   <zorder>label_36</zorder>
   <zorder>label_37</zorder>
   <zorder>valor_pinguirin</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
